docs/articles/expensify-classic/expensify-card/Auto-Reconciliation.md

```diff
@@ -35,6 +35,8 @@ To set up your auto-reconciliation account with the Expensify Card, follow these
 5. Head to the "Settings" tab.
 6. Select the account in your accounting solution that you want to use for reconciliation. Make sure this account matches the settlement business bank account.
 
+![Insert alt text for accessibility here](https://help.expensify.com/assets/images/Auto-Reconciliaton_Image1.png){:width="100%"}
```

Comment:
Can you replace all `Insert alt text for accessibility here` with a small image description?

Reply:
Oh shoot, yes I can. Doing so now.

Reply:
Hey @nkuoch is everything looking ok now?